.github/ISSUE_TEMPLATE/new-release-template.md

```diff
@@ -7,30 +7,41 @@ assignees: comcmaho, amdeel, davidmrdavid, bachuv
 
 ---
 
-**Prep Release: (assigned to:)**
-_Due: <3-business-days-before-release>_
-- [ ] Merge all features and fixes into the `dev` branch
-- [ ] Publish updated versions of DurableTask.Core and DurableTask.AzureStorage to app-service MyGet feed
-- [ ] Update dependencies and increment extension version in the `dev` branch.
-- [ ] Publish signed version of Microsoft.Azure.WebJobs.Extensions.DurableTask to app-service MyGet feed
-- [ ] Close out or punt remaining GitHub issues for the current milestone
+**Prep DTFx Release: (assigned to:)**
+_Due: <2-3-business-days-before-release>_
+- [ ] Increment updated DTFx packages (either DT-AzureStorage only or DT-AzureStorage, DT-Core, and DT-Redis)
+- [ ] Delete DTFx test packages from myget.
+- [ ] Update OneBranch durabletask repo, run signing pipeline (using these notes:  [Durable Task Framework Build Pipeline] (https://microsoft.sharepoint.com/teams/AzureWebjobs/_layouts/OneNote.aspx?id=%2Fteams%2FAzureWebjobs%2FSiteAssets%2FAzureWebjobs%20Notebook&wd=target%28Planning%2FFunctions%20Post-GA.one%7CA43CF112-7272-481A-B23E-9AA5CA8EEE06%2FDurable%20Task%20Framework%20Build%20Pipeline%7CD0946823-6FB0-44E3-A57F-E252617B69CD%2F%29
+onenote:https://microsoft.sharepoint.com/teams/AzureWebjobs/SiteAssets/AzureWebjobs%20Notebook/Planning/Functions%20Post-GA.one#Durable%20Task%20Framework%20Build%20Pipeline&section-id={A43CF112-7272-481A-B23E-9AA5CA8EEE06}&page-id={D0946823-6FB0-44E3-A57F-E252617B69CD}&end) ).
+- [ ] Publish DTFx packages to myget for testing. (either DT-AzureStorage only or DT-AzureStorage, DT-Core, DT-Redis, and DT-Emulator)
 
-
-**Validation (assigned to: )**
+**Prep Release (assigned to: )**
 _Due: <2-business-days-before-release>_
-- [ ] Run private performance tests and ensure no regressions
-- [ ] Smoke test Functions V1 and Functions V3 .NET apps
-- [ ] Smoke test JavaScript and Python apps
-- [ ] Check for package size, make sure it's not surprisingly heavier than a previous release
+- [ ] Update Durable Functions references (Analyzer? DTFx?) and increment version.
+- [ ] Add Durable Functions package to myget staging feed.
```

Comment:
Shouldn't this step happen after we merge dev into main? It is done via just launching our release pipeline, which will deploy to myget.

Reply:
No, I don't mean run signing pipeline, I mean just manually upload it to the staging myget. This step is optional, but necessary if we are each taking a smoke test/performance test role. It is just for a test package for local testing, not meant to be a signed package.

Reply:
If we are going through the process of uploading bits to myget and running smoke tests/perf tests I think we might as well use the signed release bits, and the pipeline will load it onto MyGet for us with a single button anyways, so it may even be less work. It means the exact same bits that we validated actually get loaded up to nuget.org.

Reply:
Then we would have to wait to sign off on the merge dev into main PR, run the pipeline, and solve potential roadblocks in the pipeline just to start running our first smoke test. Which might show us a new error. Then, we would be working out of dev again and merging into main etc.

This testing phase will be obsolete when the test bot is done, so we can take this out entirely when that happens and use the automated process instead.

You're saying that we are testing the same code that we are releasing, which makes sense. But we're also using the same GitHub commit so we can be sure it's the same code already.

I don't think it makes sense to require the package to be signed before running tests on it.

Reply:
I'd personally prefer to smoke testing against signed bits and deal with having to remerge into `main` if the smoke tests fail than have something subtle pop up in the difference between unsighed vs signed bits.

For instance, in the last release, we would have caught that we needed to publish a new version of DTFx.Redis + DTFx.Emulator earlier if we were using signed bits, as I believe that compiler issue only pops up with signed packages/bits.

In general, smoke tests failures should be exceedingly rare, and so I'd rather just to all of our validation on the exact package in myget so we can just do the "push package to nuget" step once validation is complete.

Reply:
> For instance, in the last release, we would have caught that we needed to publish a new version of DTFx.Redis + DTFx.Emulator earlier if we were using signed bits, as I believe that compiler issue only pops up with signed packages/bits.

What was the issue you encountered? There shouldn't be any difference at all between signed bits and unsigned bits, assuming we're talking about ESRP authenticode signing. My only concern about testing with signed bits is that it further increases the time to complete release validation. I don't think we've ever done testing with signed bits.

Reply:
I think the issue here was more related to release vs debug builds, vs signed vs unsighed builds now that I think about it. I just know it is not super straightforward to get local Release builds from the DTFx repo, but as long as we test against Release, I'm willing to budge on the signed bits.